src/routes/accommodation.route.js

```diff
@@ -118,4 +118,69 @@ router.get('/:accomodationId', verifyToken.headerToken, verifyUser, Accommodatio
  *            type: string
  */
 router.post('/booking', Validate.bookingValidation(), isValid, verifyToken.headerToken, AccomodationMiddleware.validateDates, AccomodationMiddleware.checkBookingFacilitiesAvailability, Accommodation.bookAccommodation);
+router.post('/rate/:accomodationId', verifyToken.headerToken, verifyUser, AccomodationMiddleware.checkValidAccomodationRates, AccomodationMiddleware.checkUserAccomodationCorrespondance);
+/**
+ * @swagger
+ *
+ * /accommodations/rate/{accomodationId}:
```

Comment:
@mujohn26 Always the collection is followed by the elementID. Therefore, this should be /accommodations/accommodationID/rate

Reply:
I finished working on it.